.github/workflows/build_and_test.yml

```diff
@@ -39,6 +39,7 @@ jobs:
         filters: |
           tket:
             - '{tket/src/**,tket/include/**,tket/conanfile.py,tket/CMakeLists.txt,tket/cmake/**,tket/test/**,tket/proptest/**}'
+            - '.github/workflows/build_and_test.yml'
           doxyfile:
             - '{tket/Doxyfile}'
           pytket:
```

Comment:
also add here (github won't let me suggest it)

Reply:
I mean add `- '.github/workflows/build_and_test.yml'` to the doxyfile and pytket filters as well. I just don't get github.